.planning/phases/11-self-assessment-persistence-topics/11-02-PLAN.md

```
@@ -69,306 +66,869 @@ Output: Enhanced template files that produce interactive explorers with persiste
+# explorer.ts compiles
+bun build src/explorer/template/explorer.ts --format iife --target browser --outdir /tmp/verify-11-02 2>&1
+
+# Template injection points preserved
+grep -c "__INJECT_" src/explorer/template/template.html  # Must be >= 3
+
+# Feature count check (key patterns)
+echo "Features present:"
+for pattern in "KNOWLEDGE_STATES" "localStorage" "LEARNING_MODES" "clipboard" "fallbackCopy" "migrateKnowledgeState" "debouncedSave" "exportStateAsJSON" "updateProgress" "restoreState" "mode-selector" "touch-action" "progress-fill" "max-width: 768px"; do
+  grep -rq "$pattern" src/explorer/template/ && echo "  $pattern: YES" || echo "  $pattern: NO"
+done
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)